test/tests.spec.ts

```diff
@@ -2698,16 +2606,12 @@ describe("passport-saml /", function () {
         const samlObj = new SAML(samlConfig);
 
         fakeClock = sinon.useFakeTimers(Date.parse("2014-05-28T00:13:09Z"));
-
-        samlObj.validatePostResponse(container, function (err, profile, logout) {
-          try {
-            should.exist(err);
-            err!.message!.should.match("InResponseTo is not valid");
-            done();
-          } catch (err2) {
-            done(err2);
-          }
-        });
+        try {
+          await samlObj.validatePostResponseAsync(container);
+          false.should.eql(true, "Should not succeed");
```

Comment:
Isn't there an assert that says something should throw?

Reply:
`import assert = require("assert");`
`assert.doesNotReject(samlObj.validatePostResponseAsync(container));`

Reply:
`assert.rejects(samlObj.validatePostResponseAsync(container), /Invalid signature/);`